tdms/src/iterator.cpp

```diff
@@ -743,7 +727,8 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
     if( num_fields==6 ){
       for(int i=0;i<num_fields;i++){
 	element = mxGetField( (mxArray *)prhs[input_counter], 0, C_elements[i]);
-      
+  string element_name = C_elements[i];

```

Comment:
Indentation is off

Reply:
trying to minimise the diff but will change 👍🏼